comp/virtual-comp/plan.yaml

```diff
@@ -14,13 +14,12 @@ description: >-
   computer with a discrete GPU.
 
 
-  Once the simulation is complete, the videos need to be sent to the livestream
-  producer so that they can be played during the livestream.
+  We need to plan how we'll run the virtual competition and potentially have a
+  run-through to check the setup works.
 
 
   Required actions:
    - [ ] Decide who will run the simulation
    - [ ] Decide when the simulations will be run
    - [ ] Generate a schedule for the simulated matches
```

Comment:
I'd probably move 'Generating the schedule' to running the virtual comp as this is something you can only do after deadlines have been submitted

Reply:
Alternatively, this is "Generate many suitable schedules for the simulated matches", with schedules with the desired number of appearances and a variety of team numbers